modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionServiceImpl.java

```diff
@@ -838,33 +838,38 @@ private CompletableFuture<Void> close(boolean cancel) {
                                 : closeLocalFragments();
 
                         var finalStepFut = cancelResult.whenComplete((r, e) -> {
+                            if (e != null) {
+                                Throwable ex = unwrapCause(e);
+
+                                LOG.warn(format("Fragment closing processed with errors, root fragmentId={}", rootFragmentId), ex);
```

Comment:
I've got a few questions regarding this line of code:
1) why don't delegate formatting to logger?
2) how did you come up with rootFragmentId param?
3) have you read logging guidelines?

Reply:
ok, i change it